common/src/main/scala/com/gu/media/aws/CrossAccountAccess.scala

```diff
@@ -0,0 +1,22 @@
+package com.gu.media.aws
+
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.auth.{AWSCredentialsProvider, AWSCredentialsProviderChain, STSAssumeRoleSessionCredentialsProvider}
+import com.typesafe.config.Config
+
+trait CrossAccountAccess { this: AwsAccess =>
+  def getCrossAccountCredentials(sessionId: String): AWSCredentialsProvider = {
+    new AWSCredentialsProviderChain(
+      new ProfileCredentialsProvider("composer"),
+      new STSAssumeRoleSessionCredentialsProvider(credsProvider, getStsRole(config), sessionId)
+    )
+  }
+
+  private def getStsRole(config: Config): String = {
+    if(config.hasPath("aws.kinesis.stsRoleToAssume")) {
```

Comment:
This is a raw typesafe config rather than a play config. If I don't do the check before it throws an exception